examples/src/main/scala/io/finch/wrk/Finch.scala

```diff
@@ -16,5 +19,10 @@ import io.finch.circe._
   *   - c = t * n * 1.5
   */
 object Finch extends Wrk {
-  serve(Endpoint[IO].lift(Payload("Hello, World!")).toServiceAs[Application.Json])
+
+  implicit val ioRunTime: IORuntime = IORuntime.global
+
+  Dispatcher[IO].use { implicit dispatcher =>
```

Comment:
We wouldn't need to do that if stop requiring an implicit Dispatcher. It would be easier to interact with Endpoint API that way.